### DISCUSSION TITLE
Encountered an error of "execute sql error: datasource plugin 'doris' is not found". How to fix it ?

### DISCUSSION THREAD

**Discussion by TakeYouSee** [apache/dolphinscheduler#15056] [Q&A]
The version of Dophinscheduler is 3.2.0. I would like to insert some data into another table in doris by Dolphinscheduler dispatcher, , but I got the following errer while i run a workflow with datasource type as doris. 
<img width="596" alt="1" src="https://github.com/apache/dolphinscheduler/assets/148444131/567ad2c1-0410-4bc4-bd9e-fd941331e20e">
<img width="856" alt="2" src="https://github.com/apache/dolphinscheduler/assets/148444131/6f3db0d1-7487-43f7-9ffd-d3ceb9b29a5f">

Thank you very much!

我的dolphinscheduler 版本为3.2.0。 我想要通过dohpinscheduler调度使得 apache doris 中的某一张表的数据插入到另外一张表里。工作流的组件用的是 sql， 在组件里对数据源类型的配置如上述第一张图。运行后报错显示“数据源插件‘doris’ 没有被发现” ( 曹错信息在上述第二章图中 )，请问有什么办法解决这个问题么？

**Comment by echobigdata** [comment]
请问这个解决了吗，我也遇到了同样的问题

**Comment by soltex** [comment]
请问解决了么，遇到同样问题

**Comment by mezhangremoterepository** [comment]
Change your Doris data source to MYSQL, your IP, port write your Doris configuration, so you can use mysql jdbc connection Doris

**Reply by ygq010718** [reply]
谢谢你的回复，真的的解决了我的问题，谢谢你了老弟

**Comment by CHINAJAVABoy** [comment]
Doris和MySQL高度兼容，要么使用MySQL连接，要么在数据源连接中扩展Diros代码
